src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -11,48 +11,67 @@
 
 namespace Symfony\Component\Filesystem\Tests;
 
+use Symfony\Component\Filesystem\Tests\Fixtures\StringishObject;
+
 /**
  * Test class for Filesystem.
  */
 class FilesystemTest extends FilesystemTestCase
 {
-    public function testCopyCreatesNewFile()
+    public function provideSourceTargetPaths()
     {
-        $sourceFilePath = $this->workspace.DIRECTORY_SEPARATOR.'copy_source_file';
-        $targetFilePath = $this->workspace.DIRECTORY_SEPARATOR.'copy_target_file';
+        $workspace = $this->createWorkspace();
```

Comment:
this is bad, because it will create the workspace with a different instance than the one running the test, and so the workspace will not be cleaned properly